### ISSUE TITLE
news text: gray -> black

### ISSUE THREAD

**Issue by MattHJensen** [OpenSourcePolicyCenter/webapp-public#64]
Could we make the default text color for news postings black? 
I think this is a bit hard to read: http://news.ospc.org/welcome-to-ospc.html

**Comment by hdoupe** [comment]
@MattHJensen is this still a concern?

**Comment by MattHJensen** [comment]
Yes, this is still a concern. Thanks @hdoupe. I believe the blog text would be much easier to read if it were black instead of gray so there would be higher contrast with the background. Right now we have gray on gray. 

![image](https://user-images.githubusercontent.com/8301092/33330207-3e32a3f2-d42c-11e7-9403-85979e94c685.png)
